fetch.js

```diff
@@ -1,21 +1,31 @@
 const q = require(`sql-concat`)
-const camelize = require(`camelize`)
+const camelcaseKeys = require(`camelcase-keys`)
+
+const columns = [
```

Comment:
I'd lean towards naming this `informationSchemaColumns` just to avoid the collision with that other `columns` variable on line 24